src/main/java/parking/repositories/LogRepositoryImpl.java

```diff
@@ -19,11 +18,11 @@ public LogRepositoryImpl(MongoOperations operations) {
         this.operations = operations;
     }
 
-    public void insertActionLog(ActionType actionType, ObjectId targetUserId, Integer lotNumber, Date from, Date to, LogMetaData metaData, ObjectId userId, String channel) {
+    public void insertActionLog(ActionType actionType, Account targetUser, Integer lotNumber, Date from, Date to, LogMetaData metaData, Account user, String channel) {
```

Comment:
You could also write a [javadoc](https://www.wikiwand.com/en/Javadoc) for methods with more than a few parameters or short variable names since I actually don't understand what channel variable is all about.